lib/lookup.json

```diff
@@ -421,20 +351,6 @@
     "maintainers": "ljharb",
     "scripts": ["tests-only", "posttest"]
   },
-  "rewire": {
```

Comment:
I looked into this: there is no issue with Node 20. [The test passes on GitHub](https://github.com/jhnns/rewire/actions/runs/5849412579/job/15857710898). The tests have been working fine for a couple of years. The test is also not asynchronous or contains any randomness.

[The failing test is testing the TypeScript integration](https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/3213/nodes=ubuntu1804-64/testReport/junit/(root)/citgm/rewire_v7_0_0/), which took 300ms on my machine and takes 1.2s on GitHub. My assumption is that the test timeout of 2s is too aggressive for more expensive tests in combination with a busy CPU.

What's the recommendation in that case? How can I address this? :)

Reply:
Any recommendation? :)

Reply:
I think we can use the `"script"` setting in CITGM to pass a longer timeout via CLI to `mocha`. Like maybe `npm run test --timeout 5000` will work?

Reply:
I'll try that and run CITGM and see what happens.

Reply:
Yes, passing `npm run test -- --timeout 5000` works. I've just tested it :)